src/components/Pagination/__tests__/Item.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
```

Comment:
the prop `to` is required in your component - rightly so.
your tests here should have it set in the default props
```
const defaultProps = {
  to: '/'
}
```